theano/tensor/nnet/tests/test_abstract_conv.py

```diff
@@ -1548,3 +1549,39 @@ def test_compare_1D_and_2D_upsampling_values(self):
         f_1D = theano.function([], mat_1D, mode=self.compile_mode)
         f_2D = theano.function([], mat_2D, mode=self.compile_mode)
         utt.assert_allclose(f_1D(), f_2D(), rtol=1e-06)
+
+
+class TestConv2dTranspose(unittest.TestCase):
+    mode = None
```

Comment:
This should probably be something like `theano.compile.get_default_mode()` or `theano.compile.get_mode(None)`.